libexec/tgenv-list-remote

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 set -e
+pages=4
```

Comment:
Any reason this is specifically 4? Wouldn't this mean we'd need to change this value in the future?

Reply:
we might. for now it works, and it is the maximal number of pages. tried finding the max page, needs working with regexes, over the headers of the request. burnt too much time on it. defaulted to magic number for now, willing to open a ticket for later fixing it